### PR TITLE
Add hcloud Ansible Tutorial

### PR DESCRIPTION
**Pull request by LKaemmerling**

### CODE REVIEW

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+path: "/tutorials/howto-hcloud-ansible"
+slug: "howto-hcloud-ansible"
+date: "2019-05-02"
+title: "How-to: Hetzner Cloud Ansible modules"
+short_description: "How to use the Hetzner Cloud Ansible modules for creating and managing resources on the Hetzner Cloud."
+tags: ["Hetzner Cloud", "hcloud", "Ansible"]
+author: "Lukas Kämmerling"
+author_ink: "https://github.com/LKaemmerling"
+author_img: "https://avatars1.githubusercontent.com/u/4281581?s=400&v=4"
```
Comment (raspla) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
author_img: "https://avatars1.githubusercontent.com/u/4281581"
```

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          server_type: cx11
+          image: ubuntu-18.04
+          state: present
+      register: server
+    - name: Create a volume
+      hcloud_volume:
+          api_token: "{{ hcloud_token }}"
+          name: my-volume
+          size: 10
+          server: "{{ server.hcloud_server.name}}"
```
Comment (raspla) [inline tutorials/howto-hcloud-ansible/01.en.md]: Missing space at the end of the line?

**Review by raspla — COMMENTED** [review]
Looks good to me so far. Maybe use always `Ansible`, sometimes you have written lowercased `ansible`.

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+## Introduction
+
+In this how-to you will learn to use the hcloud Ansible modules. This includes how to create and delete resources and how to do more complex scenarios like attaching a volume to a server.
+
+ **Prerequisites**
+
+ * Hetzner Cloud API Token 
+
+   * Basic knowledge about the Hetzner Cloud
+       * We assume that you know what a server, an image, a server type and a volume is.
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
       * knowing what a server, an image, a server type and a volume is.
```

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  * Visit Hetzner Cloud Console at [https://console.hetzner.cloud](https://console.hetzner.cloud), select your project, and create a new API Token.
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]: I'm a bit confused here, the first point is `Hetzner Cloud API Token` and here you give instructions on how to create one. Wouldn't that be a underpoint while the knoweledge thats required should have it's own top level point?

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+* [Ansible 2.8](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) is installed and you have basic knowledge about it.
+* `hcloud-python` installed
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
* `hcloud-python` installed (`pip install hcloud`)
```

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+   * Just run `pip install hcloud`
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
```

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+## Step 1 - Basic Usage
+
+In Ansible, you can control your infrastructure by using yaml files, which describe the state of your infrastructure. These files are called "Roles". Every role can have tasks. A task is something like "create a server", "run a command on the server" or something else. You can control ansible with the `ansible` command.
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
In Ansible you can control your infrastructure with yaml files. They describe the state of your infrastructure. These files are called "Roles". Every role can have tasks. A task is something like "create a server", "run a command on the server" or something else. You can control ansible with the `ansible` command. 
```

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+   * Basic knowledge about the Hetzner Cloud
+       * We assume that you know what a server, an image, a server type and a volume is.
+  * Visit Hetzner Cloud Console at [https://console.hetzner.cloud](https://console.hetzner.cloud), select your project, and create a new API Token.
+* [Ansible 2.8](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) is installed and you have basic knowledge about it.
+* `hcloud-python` installed
+   * Just run `pip install hcloud`
+
+## Step 1 - Basic Usage
+
+In Ansible, you can control your infrastructure by using yaml files, which describe the state of your infrastructure. These files are called "Roles". Every role can have tasks. A task is something like "create a server", "run a command on the server" or something else. You can control Ansible with the `ansible` command.
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
In Ansible, you can control your infrastructure by using yaml files. They describe the state of your infrastructure. These files are called "Roles". Every role can have multiple tasks. A task is something like "create a server", "run a command on the server" or something else. You can control Ansible with the `ansible` command. 
```

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+  * Visit Hetzner Cloud Console at [https://console.hetzner.cloud](https://console.hetzner.cloud), select your project, and create a new API Token.
+* [Ansible 2.8](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) is installed and you have basic knowledge about it.
+* `hcloud-python` installed
+   * Just run `pip install hcloud`
+
+## Step 1 - Basic Usage
+
+In Ansible, you can control your infrastructure by using yaml files, which describe the state of your infrastructure. These files are called "Roles". Every role can have tasks. A task is something like "create a server", "run a command on the server" or something else. You can control Ansible with the `ansible` command. 
+
+For usage of the module, you only need to create a task like the following:
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
To use the module a taske like the following is all you need:
```

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+      api_token: "YOUR_API_TOKEN"
+      name: my-server
+      server_type: cx11
+      image: ubuntu-18.04
+      state: present
+```
+All Hetzner Cloud modules are built into Ansible 2.8!
+
+## Step 2 - Create a server
+So you have learned something about the basic usage of Ansible. Now we will show you, how you can create a new server with the hcloud-server module. First of all, you should save the following YAML as `hcloud-server.yml`.
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
You have learned something about the basic usage of Ansible. Now we will show you, how you can create a new server with the hcloud-server module. First of all, you should save the following YAML as `hcloud-server.yml`.
```

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+          api_token: "{{ hcloud_token }}"
+          name: my-server
+          server_type: cx11
+          image: ubuntu-18.04
+          state: present
+      register: server
+```
+
+The snippet will create a new server called `my-server` with the server type `cx11` and the image `ubuntu-18.04`, the state is `present` so the module will create the server.
+When you now run `ansible-playbook -v hcloud-server.yml` you should get a output similar to this:
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
When you now run `ansible-playbook -v hcloud-server.yml` you should get a output similar to this:
```

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+
+TASK [Create a volume] **************************************************************************************************************************************************************************************************************************************
+changed: [localhost] => {"changed": true, "hcloud_volume": {"id": "2489399", "labels": {}, "location": "nbg1", "name": "my-volume", "server": "my-server", "size": 10}}
+
+PLAY RECAP **************************************************************************************************************************************************************************************************************************************************
+localhost                  : ok=2    changed=2    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+```
+
+You have created a server and a volume which is attached to the server!
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
You have created a server and an attached volume! 
```

File: tutorials/howto-hcloud-ansible/01.en.md [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+
+PLAY RECAP **************************************************************************************************************************************************************************************************************************************************
+localhost                  : ok=2    changed=2    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+```
+
+You have created a server and a volume which is attached to the server! 
+
+## Conclusion
+
+You now have a basic overview of the Hetzner Cloud Ansible modules. We have covered how you can create and delete resources and how to do more complex scenarios like attaching a volume to a server. You can find more help on the [official documentation](https://docs.ansible.com/ansible/latest/modules/list_of_cloud_modules.html#hcloud). If you need deeper help, just open an issue on our [Github Repository](https://github.com/ansible/ansible).
```
Comment (frommMoritz) [inline tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
You now have a basic overview of the Hetzner Cloud Ansible modules. We covered how you can create and delete resources and how to do more complex scenarios like attaching a volume to a server. You can find more help on the [official documentation](https://docs.ansible.com/ansible/latest/modules/list_of_cloud_modules.html#hcloud). If you need further help, just open an issue on our [Github Repository](https://github.com/ansible/ansible).
```

**Review by frommMoritz — CHANGES_REQUESTED** [review]

File: tutorials/howto-hcloud-ansible/01.en.md (hunk 9 above)
Comment (LKaemmerling) [reply tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
To use the module, a task like the following is needed:
```

File: tutorials/howto-hcloud-ansible/01.en.md (hunk 11 above)
Comment (LKaemmerling) [reply tutorials/howto-hcloud-ansible/01.en.md]:
```suggestion
When you run `ansible-playbook -v hcloud-server.yml` you should get an output similar to this below:

File: tutorials/howto-hcloud-ansible/01.en.md (hunk 4 above)
Comment (LKaemmerling) [reply tutorials/howto-hcloud-ansible/01.en.md]: I have changed it a bit, thank you!

**Pull request closed without merging**